### PR TITLE
StrongName Sign assemblies before Authenticode signing

### PR DESCRIPTION
**Pull request by vijayrkn**
@joeloff

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -185,19 +185,24 @@
...
+    </Target>
+    
   <!-- Localization -->
   <Target Name="BuildLocalized" DependsOnTargets="SetLanguages" Condition="Exists('.tools\loc\loc.proj')">
     <MSBuild Projects=".tools\loc\loc.proj" Targets="Build"
              Properties="Lang=%(Languages.Identity)" />
-			 	
-	<MSBuild Condition="'$(PB_SignType)' == 'real' or '$(PB_SignType)' == 'test'"
+
+    <!-- String name the localized assemblies-->
```
Comment (joeloff) [inline build.proj]: Typo :) should be Strongname, not String name

Comment (vijayrkn) [reply build.proj]: oops. Will fix it.

**Comment by vijayrkn** [comment]
yes, strong name signing happens as part of build and signing happens after that.

**Comment by joeloff** [comment]
:shipit:

**Pull request merged**